vulkaninfo/generated/vulkaninfo.hpp

```diff
@@ -1988,7 +1988,9 @@ void DumpVkPhysicalDeviceHostImageCopyPropertiesEXT(Printer &p, std::string name
     ObjectWrapper object{p, name};
     p.SetMinKeyWidth(35);
     p.PrintKeyValue("copySrcLayoutCount", obj.copySrcLayoutCount);
-    {
+    if (obj.copySrcLayoutCount == 0) {
+        p.PrintKeyValue("pCopySrcLayouts", "nullptr");
```

Comment:
I have a slight preference for `"NULL"` since that is what the spec uses

Reply:
That makes more sense, updated.